src/pps/pps.proto

```diff
@@ -670,15 +670,14 @@ message ListDatumRequest {
   bool reverse = 6;
 }
 
-// Emits a stream of datums as they are created from the given input. Client
```

Comment:
Is this comment no longer relevant?

Reply:
It's still true. The RPC signature makes this clear so removed the comment.

Reply:
If it's still true I'd leave it in. I think "Client must cancel the stream when it no longer want to receive datums" is information that the signature doesn't convey.